types/elsys/ers/configuration.schema.json

```diff
@@ -0,0 +1,426 @@
+{
+  "$id": "https://akenza.io/elsys/ers/configuration.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Configuration",
+  "type": "object",
+  "properties": {
+    "appSKey": {
```

Comment:
outputting keys as data does not seem to be best practice, but if it is the Elsys way we should support it

Reply:
Usually they are not sent, but they could.